### ISSUE TITLE
NPC: Shenthul

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#68]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Orgrimmar / Cleft of Shadow / NPC: Shenthul
Link: https://www.wowhead.com/wotlk/npc=3401/shenthul

Greeting: the voice-over suddenly silences during playback.

Dual Specialization text: the voice-over suddenly silences during playback.

**Comment by carevass** [comment]
This should be fixed in AI_VoiceOverData_VanillaExtra_v1.5.
